src/main/java/org/owasp/esapi/reference/DefaultSecurityConfiguration.java

```diff
@@ -581,13 +582,17 @@ public File getResourceFile(String filename) {
         }
 
 		if (fileUrl != null) {
-			String fileLocation = fileUrl.getFile();
-			f = new File(fileLocation);
-			if (f.exists()) {
-				logSpecial("Found in SystemResource Directory/resourceDirectory: " + f.getAbsolutePath());
-				return f;
-			} else {
-				logSpecial("Not found in SystemResource Directory/resourceDirectory (this should never happen): " + f.getAbsolutePath());
+			try {
+				String fileLocation = fileUrl.toURI().getPath();
+				f = new File(fileLocation);
```

Comment:
@kwwall at what point do we want to worry about changing our file calls with *.nio?  [Clearly not on this PR, just wondering if we want to bucket that for 3.0?]